content/webapp/components/GuideStopCard/index.tsx

```diff
@@ -0,0 +1,115 @@
+import { FunctionComponent } from 'react';
+import { getCrop, ImageType } from '@weco/common/model/image';
+import PrismicImage from '@weco/common/views/components/PrismicImage/PrismicImage';
+import Icon from '@weco/common/views/components/Icon/Icon';
+import Space from '@weco/common/views/components/styled/Space';
+import { grid } from '@weco/common/utils/classnames';
+import { threeUpGridSizesMap } from '@weco/content/components/Body/GridFactory';
+
+import {
+  map,
+  audio as audioIcon,
+  video as videoIcon,
+} from '@weco/common/icons';
+import {
+  CardOuter,
+  CardTitle,
+  CardBody,
+  CardImageWrapper,
+} from '@weco/content/components/Card/Card';
+
+function secondsToMinutesAndSeconds(s: number): string {
+  const minutes = Math.floor(s / 60);
+  const seconds = s % 60;
+  const padSeconds = seconds < 10 ? `0${seconds}` : seconds;
+  const padMinutes = minutes < 10 ? `0${minutes}` : minutes;
+
+  return `${padMinutes}:${padSeconds}`;
+}
```

Comment:
np: worth moving to a `utils/time.ts` file like we have for dates? I wondered if we had anything already for "Reading time" but [it doesn't seem like it](content/webapp/utils/reading-time.ts), although I feel like these utils could also be moved to `utils/time.ts` should it exist.. The goal would be that if we ever needed to display a time anywhere else (probably in the audio player, right?), a dev could first look at utils to see if anything exists.

Reply:
sounds good. Maybe a unit test or two as well